src/runner.js

```diff
@@ -1,26 +1,62 @@
+const isSPDXCompliant = require('spdx-expression-validate');
+const satisfiesSPDXLicense = require('spdx-satisfies');
+
 const {
-  getPackageInfoList, extractInvalidPackages, formatForbiddenLicenseError
+  getPackageInfoList, formatForbiddenLicenseError
 } = require('./utils');
 
-const reporter = require('./reporter');
+const checkLicenseArgument = (args = []) => {
```

Comment:
@neodmy , do you find it useful to move `checkLicenseArgument`, `generateSPDXExpression` and `checkPackagesLicenses` to the `utils` folder? Or is it better to keep them here?

Reply:
Done. I've renamed some functions and their arguments to make them more _agnostic_